client/src/components/ElementEditor/ElementEditor.js

```diff
@@ -8,12 +8,25 @@ import { elementTypeType } from 'types/elementTypeType';
  */
 class ElementEditor extends PureComponent {
   render() {
-    const { ToolbarComponent, ListComponent, pageId, elementTypes, baseAddHref } = this.props;
+    const {
+      ToolbarComponent,
+      ListComponent,
+      pageId,
+      elementalAreaId,
+      elementTypes,
+    } = this.props;
 
     return (
       <div className="element-editor">
-        <ToolbarComponent elementTypes={elementTypes} baseAddHref={baseAddHref} />
-        <ListComponent elementTypes={elementTypes} pageId={pageId} baseAddHref={baseAddHref} />
+        <ToolbarComponent
+          elementTypes={elementTypes}
+          elementalAreaId={elementalAreaId}
+        />
+        <ListComponent
+          elementTypes={elementTypes}
+          pageId={pageId}
+          elementalAreaId={elementalAreaId}
+        />
```

Comment:
I generally think that these kinds of changes are unnecessary and pollute the purpose of a PR - maybe in future we could keep them to separate PRs

Reply:
Tests will fail because of line length if I don't ;)

Reply:
I see, you've added a new prop to it - disregard

Reply:
Yes, it is an annoyance of this type of change, I agree. It's hard to quickly see what has changed - but the linter be as the linter do! (except where its not defined about trailing commas... :/ )